packages/components/src/Input/_Input.test.tsx

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable testing-library/no-node-access */
+import { screen } from '@testing-library/react';
```

Comment:
why extra file for tests? @aromko

Reply:
@sarahgm because codecov is moaning when I add a new file and no tests are available.

Reply:
Ah as I understand we used for stories and test the new component anyway 🤔 or how do you mean?